# Backport upstream commit caab275509826dc5fe5ab2632582abb8f83ea2b3 to link
# against compiled Boost Filesystem library; this is not a header-only library.
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5b37161..38be82a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -42,7 +42,7 @@ else()
     set(BOOST_COMPONENTS regex)
 endif()
 
-list(APPEND BOOST_COMPONENTS thread date_time chrono system program_options)
+list(APPEND BOOST_COMPONENTS thread date_time chrono filesystem system program_options)
 
 # Add other dependencies
 find_package(Boost 1.54 REQUIRED COMPONENTS ${BOOST_COMPONENTS})
